cpp/src/arrow/filesystem/gcsfs_test.cc

```diff
@@ -119,7 +122,23 @@ class GcsIntegrationTest : public ::testing::Test {
             .set<gc::UnifiedCredentialsOption>(gc::MakeInsecureCredentials()));
   }
 
+  std::string RandomLine(int lineno, std::size_t width) {
+    auto const fillers = std::string("abcdefghijlkmnopqrstuvwxyz0123456789");
+    std::uniform_int_distribution<std::size_t> d(0, fillers.size() - 1);
```

Comment:
You can probably reuse `random_string` from `arrow/testing/util.h` instead.

Reply:
`random_string()` produces non-printable characters, which are not the easiest thing to troubleshoot.  `random_ascii()` could do it, but has a weird API using a `uint8_t*` output parameter.  I would rather leave this as-is.

Reply:
Fair enough :-)